src/pages/en.md

```diff
@@ -316,9 +316,16 @@ For example, Rails has a [`rails-dev-box`](https://github.com/rails/rails-dev-bo
 
 [Docker](https://www.docker.com) is also a good way to make dependency management simpler.
 
-- [ ] Promptly update version of gem at RubyGems
+- [ ] Don't forget to update version of gem at RubyGems
 
-You need to keep version of gem at [RubyGems](https://rubygems.org/) is up-to-date.
+You need to keep version of gem at [RubyGems](https://rubygems.org/) up-to-date. You should use this clarification when make decision about updating:
```

Comment:
"You should use" sounds too imperative to me. I'd prefer to give an advice and not an order)

So, maybe, smth like that:

>You can use the following reasonings when making a decision about the release: 
...